doc-bridgepoint/notes/9566_constants_ant.md

```diff
@@ -0,0 +1,77 @@
+---
+
+This work is licensed under the Creative Commons CC0 License
+
+---
+
+# Constant group item visibility
+### xtUML Project Analysis Note
+
+### 1. Abstract
+
+Constant groups and their items are visible across projects globally. The
+"namespace" of a constant has to be included in the name to avoid naming
+conflicts. Suggestion is to at least hide the items behind the constant group
+name similar to how enums and enum items are managed.
+
+### 2. Document References
+
+<a id="2.1"></a>2.1 [#9566 Constant group item visibility](https://support.onefact.net/issues/9566)  
+
+### 3. Background
+None
+
+### 4. Requirements
```

Comment:
I'd like to see the requirements from the SRS explicitly called out and the test section clearly show where each requirement is tested.